mlflow/server/handlers.py

```diff
@@ -1119,6 +1124,48 @@ def search_datasets_handler():
         return _not_implemented()
 
 
+@catch_mlflow_exception
+def gateway_proxy_handler():
+    args = request.json
+    host = args.get("host")
+    if not host:
+        raise MlflowException(
+            message="GatewayProxy request must specify a host.",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
+    port = args.get("port")
+    if not port:
+        raise MlflowException(
+            message="GatewayProxy request must specify a port.",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
+    gateway_path = args.get("gateway_path")
+    if not gateway_path:
+        raise MlflowException(
+            message="GatewayProxy request must specify a gateway_path.",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
+    request_type = args.get("request_type")
+    if not request_type:
+        raise MlflowException(
+            message="GatewayProxy request must specify a request_type.",
+            error_code=INVALID_PARAMETER_VALUE,
+        )
```

Comment:
If the frontend is making the request to the gateway proxy handler, how does the frontend know which gateway host, port, etc. to pass?

I think it might be simpler for the tracking server to assume that the host, port, and path have already been configured via environment variable or CLI flag when starting `mlflow server`.

Reply:
Ref: https://github.com/mlflow/mlflow/pull/9452/files#r1304866995

Reply:
Ah that is fair. Let's use environment variables to manage this.